### DISCUSSION TITLE
Privacy Focused Share This Alternative

### DISCUSSION THREAD

**Discussion by derpycoder** [plausible/analytics#705] [Anything Else]
Hey Guys,

I know you are focused on delivering higher quality for the Analytics side of things, but isn't tracking user share count also considered a useful stat?

After adding ShareThis, I was elated that people are sharing the link even if it's only 10 times.

I couldn't find a privacy focused alternative to that, and neither to the feedback component. So I was hoping if a component like that will be considered in the future.

I am willing to pay a little extra for this add on:
1. Dashboard would house share count as well and I won't have to manually visit each page to see the share count.
2. No user data will be stolen.
3. User feedback will show up in private dashboard.
4. Emotion component or Upvotes & Downvotes can be provided as well.

P.S. I will soon make the UI Equivalent of Share This & a Feedback Component. It would be awesome if I didn't have to build a Dashboard like Plausible has, from scratch!

See how it currently works in my Blog Post: [Snowtail: Snowpack & Tailwind Starter to Speed up Theme Dev](https://www.derpycoder.com/snowtail-snowpack-tailwind-starter-to-speed-up-theme-dev/)

**Comment by Vigasaurus** [comment]
I don't know about Plausible offering the UI kit for the buttons themselves, but you can easily use [custom goals](https://plausible.io/docs/custom-event-goals) to accomplish this with Plausible (assuming you bring your own buttons). As for actually bringing the share content, you could just have a watcher for hashtags or search terms on any social platform - but integrating that into Plausible seems honestly like a pretty rare use case (I could be wrong).
